.circleci/config.yml

```diff
@@ -18,23 +18,26 @@ jobs:
               pip install black --progress-bar off
               pip install "isort[pyproject]" numpy --progress-bar off
               pip install mypy==0.950 --progress-bar off
-              pip install -r requirements.txt --progress-bar off
+              pip install -r habitat-lab/requirements.txt --progress-bar off
       - run:
           name: run black
           command: |
-              black --exclude '/(\.eggs|\.git|\.hg|\.mypy_cache|\.nox|\.tox|\.venv|_build|buck-out|build|dist)|examples/tutorials/(colabs|nb_python)' habitat/. habitat_baselines/. examples/. test/. setup.py --diff
-              black --exclude '/(\.eggs|\.git|\.hg|\.mypy_cache|\.nox|\.tox|\.venv|_build|buck-out|build|dist)|examples/tutorials/(colabs|nb_python)' habitat/. habitat_baselines/. examples/. test/. setup.py --check
+              black --exclude '/(\.eggs|\.git|\.hg|\.mypy_cache|\.nox|\.tox|\.venv|_build|buck-out|build|dist)|examples/tutorials/(colabs|nb_python)' habitat-lab/. habitat-baselines/. examples/. test/. --diff
+              black --exclude '/(\.eggs|\.git|\.hg|\.mypy_cache|\.nox|\.tox|\.venv|_build|buck-out|build|dist)|examples/tutorials/(colabs|nb_python)' habitat-lab/. habitat-baselines/. examples/. test/. --check
       - run:
           name: run isort
           command: |
               isort --version
-              isort habitat/. habitat_baselines/. examples/. test/. setup.py --diff
-              isort habitat/. habitat_baselines/. examples/. test/. setup.py --check-only
+              isort habitat-lab/. habitat-baselines/. examples/. test/.  --diff
+              isort habitat-lab/. habitat-baselines/. examples/. test/.  --check-only
       - run:
           name: run mypy
           command: |
               mypy --version
-              mypy
+              mypy habitat-lab/
```

Comment:
You shouldn't need to specify any args if the mypy config is setup properly?

Reply:
Unfortunately, mypy has an issue where a repo with two setup.py files conflict with each other : https://github.com/python/mypy/issues/10428 
The error is the following : 
```
habitat-baselines/setup.py: error: Duplicate module named "setup" (also at "habitat-lab/setup.py")
```

Reply:
I could get around it by doing 
```
exclude=habitat-baselines/setup.py
```
in myy.ini, but I don't want to exclude this file from mypy